### PR TITLE
fix(input): hide label should work with axe, #36

### LINKED ISSUE #36: input hidelabel is not accessible
When using the hidelabel property the control fails axe.

### PR DESCRIPTION
**Pull request by rikbrowning**
Note we are only working with WCAG standards and ignoring best practices preset from axe

### CODE REVIEW

File: packages/@featherds/input-helper/src/components/InputWrapper.vue [hunk 1]
```diff
@@ -305,6 +307,11 @@ export default {
       }
     }
   }
+  &.hide-label {
+    label {
+      @include screen-reader;
```
Comment (jmillar-nanthealth) [inline packages/@featherds/input-helper/src/components/InputWrapper.vue:312]: This does not appear to be enough to hide the label properly, the control is overriding the `left` CSS rule and you can still see a small moving gap when focus/blur occurs
image

**Comment by jmillar-nanthealth** [comment]
In addition to the comment above, the demo pages have different spacing and it appears the `subtext` component is being rendered even with no content. The subtext component test for whether the slot inside is empty is not is not working as expected and returning `true`.
Can you verify on your machines?

**Comment by rikbrowning** [comment]
Yes subtext is being rendered but we always want it to be rendered, infact we need to set a min-height to 1.5rem on it so spacing is always consistent, will do that in another ticket

**Comment by jmillar-nanthealth** [comment]
We may want to have it always rendered, but the issue here is that the code to detect content in the slot no longer works and may have impacts in other parts of the codebase. Something has recently changed and may be causing issues. Mostly just a heads-up.

**Review by jmillar-nanthealth — APPROVED** [review]

**Pull request merged**